README.md

```diff
@@ -1015,113 +1103,157 @@ Constructs equi-width histograms, in which the histogram range is divided into i
 Notes:
 
 **Args:**
+
 - `expr`: The expression for which the histogram is created. This expression must evaluate to a numeric value or to a value that can be implicitly converted to a numeric value.
 
 - `min_value` and `max_value`: The low and high end points of the acceptable range for the expression. The end points must also evaluate to numeric values and not be equal.
 
 - `num_buckets`:  The desired number of buckets; must be a positive integer value. A value from the expression is assigned to each bucket, and the function then returns the corresponding bucket number.
 
 When an expression falls outside the range, the function returns:
+
 - `0` if the expression is less than min_value.
 - `num_buckets + 1` if the expression is greater than or equal to max_value.
 
-
 **Usage:**
+
 ```
 {{ dbt_utils.width_bucket(expr, min_value, max_value, num_buckets) }}
 ```
 
 ### Web macros
+
 #### get_url_parameter ([source](macros/web/get_url_parameter.sql))
+
 This macro extracts a url parameter from a column containing a url.
 
 **Usage:**
+
 ```
 {{ dbt_utils.get_url_parameter(field='page_url', url_parameter='utm_source') }}
 ```
 
 #### get_url_host ([source](macros/web/get_url_host.sql))
+
 This macro extracts a hostname from a column containing a url.
 
 **Usage:**
+
 ```
 {{ dbt_utils.get_url_host(field='page_url') }}
 ```
 
 #### get_url_path ([source](macros/web/get_url_path.sql))
+
 This macro extracts a page path from a column containing a url.
 
 **Usage:**
+
 ```
 {{ dbt_utils.get_url_path(field='page_url') }}
 ```
+
 ----
+
 ### Cross-database macros
+
 These macros make it easier for package authors (especially those writing modeling packages) to implement cross-database
 compatibility. In general, you should not use these macros in your own dbt project (unless it is a package)
 
+Note that most of these macros have now moved to dbt-core. Instead of the `dbt_utils.` namespace, you should now call these macros with the `dbt.` prefix, or without a prefix altogether (as shown in the examples below). They will eventuall be removed from `dbt_utils`.
```

Comment:
```suggestion
Note that most of these macros moved to dbt Core as of dbt_utils v0.9.0 and dbt Core v1.2.0, and will soon be removed from `dbt_utils`. 

To access the version defined in dbt Core, remove the `dbt_utils.` prefix (see [https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros](https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros) for examples). 
```

I think that since this README is also shown to people who are using old versions, we may want to keep the old documentation that includes the `dbt_utils.` prefix. Otherwise people using an older version of utils who follow this advice will get an error message.